todo/src/components/TodoList.js

```diff
@@ -38,16 +109,78 @@ function TodoList(props) {
 				</form>
 			</div>
 			<div
-				style={{ border: "1px solid black", height: "100%", display: "relative" }}
+				style={{
+					height: "100%",
+					display: "relative",
+				}}
 			>
-				{todos.map((todo, index) => (
-					<TodoItem
-						key={uuid()}
-						onClick={handleSubmit}
-						content={todo.content}
-						index={index}
-					></TodoItem>
-				))}
+				<DragDropContext onDragEnd={onDragEnd}>
+					<Droppable droppableId="task-list">
+						{(provided) => (
+							<ul ref={provided.innerRef} {...provided.droppableProps}>
+								{filteredTasks.map((task, index) => (
+									<Draggable key={task.id} draggableId={task.id} index={index}>
+										{(provided) => (
+											<li
+												ref={provided.innerRef}
+												{...provided.draggableProps}
+												{...provided.dragHandleProps}
+											>
+												<TodoItem
+													darkModeEnabled={props.darkModeEnabled}
+													index={index}
+													key={task.id}
+													onClick={deleteTask}
+													content={task.content}
+													cb={task.isComplete}

```

Comment:
What does "cb" mean?

Reply:
it means check box i fixed it to be more clear